hieradata/common.yaml

```diff
@@ -339,7 +339,7 @@ govuk::apps::bouncer::nagios_memory_warning: 1400
 govuk::apps::bouncer::nagios_memory_critical: 1500
 govuk::apps::bouncer::unicorn_worker_processes: "8"
 
-govuk::apps::cache_clearing_service::enabled: true
+govuk::apps::cache_clearing_service::enabled: false
```

Comment:
Because the govuk-puppet development-vm uses the Carrenza hieradata, it might be good to keep the `enabled: true` value in `development.yaml`, so that the service is still setup in the development vm.